Sources/PT.PM.Common/TextUtils.cs

```diff
@@ -78,42 +73,54 @@ public static TextSpan ParseAnyTextSpan(string textSpanString, out bool isLineCo
             return result;
         }
 
-        public static TextSpan ParseTextSpan(string text, TextFile currentSourceFile = null, HashSet<IFile> sourceFiles = null)
+                public static TextSpan ParseTextSpan(string text, TextFile currentSourceFile = null, HashSet<IFile> sourceFiles = null)
         {
-            string[] parts = text.Split(semicolon, 2);
+            ReadOnlySpan<char> textSpan = text.AsSpan();
+            int semicolonIndex = textSpan.IndexOf(';');
 
-            string fileName = parts.Length == 2
-                ? parts[1].Trim()
-                : null;
+            ReadOnlySpan<char> location;
+            ReadOnlySpan<char> fileName;
+
+            if (semicolonIndex >= 0)
+            {
+                location = textSpan.Slice(0, semicolonIndex).Trim();
+                fileName = textSpan.Slice(semicolonIndex + 1).Trim();
+            }
+            else
+            {
+                location = textSpan.Trim();
+                fileName = ReadOnlySpan<char>.Empty;
+            }
 
             TextFile sourceFile = (TextFile)GetSourceFile(fileName, currentSourceFile, sourceFiles);
 
             TextSpan result;
             try
             {
-                string range = parts[0].Trim().Substring(1, parts[0].Length - 2);
-                int index = range.IndexOf("..");
+                ReadOnlySpan<char> range = location.Slice(1, location.Length - 2);
+                int index = range.IndexOf("..".AsSpan());
 
                 int start, end;
                 if (index != -1)
                 {
-                    string value = range.Remove(index);
+                    string value = range.Slice(0, index).ToString(); // TODO: It will be replaced on Span when netstandard2.1 comes
```

Comment:
Will be replaced *with* Span, you mean?

"when netstandard2.1 comes _out_" I believe...